docs/docs/feature/encoders.md

```diff
@@ -25,7 +25,7 @@ Rotation is handled separately as a type of sensor. The behavior for this is set
 sensor-bindings = <BINDING CW_KEY CCW_KEY>;
 ```
 
-- `BINDING` is one of two rotation bindings that are currently defined, `&inc_dec_cp` for consumer key presses or `&inc_dec_kp` for normal key presses (see [Key Press](/docs/behavior/key-press) for the difference between the two).
+- `BINDING` is currently only the one implemented bindings; `&inc_dec_kp` for key presses (see [Key Press](/docs/behavior/key-press) for details on available keycodes).
```

Comment:
> only the one implemented bindings
Have you had your coffee today?

Reply:
Not enough, clearly.

Reply:
Yeah, updated. Not sure if my tired brain wrote copy that's significantly better. Thoughts?

Reply:
I think the approach here is to rephrase the stuff around it, e.g.

`BINDING`: the only relevant behavior currently implemented is `&inc_dec_kp`.
`CW_KEY`: the keycode ...
`CCW_KEY`: the keycode ...

Reply:
Scratch that!  GitHub was showing me an outdated version.